### ISSUE TITLE
False positive, ```type mapping mismatch: database can contain string but property expects int```

### ISSUE THREAD

**Issue by ErnestoNCarrea** [phpstan/phpstan-doctrine#479]
SQL bigint is incorrectly recognized as string. This code, which uses bigint for a column:

```php
#[ORM\Column(name: "id", type: "bigint")]
private int $id;
```

produces the following false positive:

```Property (class)::$id type mapping mismatch: database can contain string but property expects int.```

**Comment by ErnestoNCarrea** [comment]
This is Symfony and Doctrine, by the way. Latest available versions of both, and phpstan.

**Comment by ondrejmirtes** [comment]
This is correct. Doctrine maps bigint into string. Check out the documentation.
